src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -172,6 +175,25 @@ internal static class DefinitionItemExtensions
             return properties;
         }
 
+        private static ImmutableArray<AdditionalProperty> GetDisplayableProperties(ISymbol definition)
+        {
+            var displayableProperties = ImmutableArray<AdditionalProperty>.Empty;
+
+            var containingType = definition.ContainingType;
+            if (containingType != null)
+            {
+                displayableProperties = displayableProperties.Add(new AdditionalProperty(AbstractReferenceFinder.s_containingTypeInfo, containingType.Name));
+            }
+
+            var containingSymbol = definition.ContainingSymbol;
+            if (containingSymbol != null && !containingSymbol.IsNamespace())
```

Comment:
Talked to @chborl offline - we should be filtering this further based on symbol kind (method/event/property/field) to ensure containing member column only has entries for member symbols.